src/components/Container/Container.tsx

```diff
@@ -5,6 +5,10 @@ interface Props {
    children: React.ReactNode;
 }
 
-export const Container: React.FC = (props: Props) => (
-   <div className={styles.container}>{props.children}</div>
-);
+// export const Container: React.FC = (props: Props) => (
+//    <div className={styles.container}>{props.children}</div>
+// );
+
+export const Container: React.FC<Props> = ({ children }) => {
+   return <div className="container">{children}</div>;
```

Comment:
**nitpick:** It should be CSS module here, instead of global style - previous version was correct